README.md

```diff
@@ -67,43 +67,36 @@ NB: you can also download MeiliSearch from **Homebrew** or **APT**.
 package main
 
 import (
-    "fmt"
-    "os"
+	"fmt"
+	"os"
 
-    "github.com/meilisearch/meilisearch-go"
+	"github.com/meilisearch/meilisearch-go"
 )
 
 func main() {
-    var client = meilisearch.NewClient(meilisearch.Config{
-        Host: "http://127.0.0.1:7700",
-        APIKey: "masterKey",
-    })
-
-    // Create an index if your index does not already exist
-    _, err := client.Indexes().Create(meilisearch.CreateIndexRequest{
-        UID: "books",
-    })
-    if err != nil {
-        fmt.Println(err)
-        os.Exit(1)
-    }
-
-    documents := []map[string]interface{}{
-        {"book_id": 123,  "title": "Pride and Prejudice"},
-        {"book_id": 456,  "title": "Le Petit Prince"},
-        {"book_id": 1,    "title": "Alice In Wonderland"},
-        {"book_id": 1344, "title": "The Hobbit"},
-        {"book_id": 4,    "title": "Harry Potter and the Half-Blood Prince"},
-        {"book_id": 42,   "title": "The Hitchhiker's Guide to the Galaxy"},
-    }
-
-    updateRes, err := client.Documents("books").AddOrUpdate(documents) // => { "updateId": 0 }
-    if err != nil {
-        fmt.Println(err)
-        os.Exit(1)
-    }
-
-    fmt.Println(updateRes.UpdateID)
+	client = meilisearch.NewClient(meilisearch.ClientConfig{
```

Comment:
Shouldn't `client` exist already (previously defined/initialized) or use := for this to work?